### ISSUE TITLE
Running tests in SBT

### ISSUE THREAD

**Issue by mattinbits** [jbrisbin/akka-http-docker#3]
When I run 'sbt test' the code compiles correctly but no tests are run. 

```
sbt test
[info] Loading project definition from /Users/matt/code/akka-http-docker/project
[info] Set current project to akka-http-docker (in build file:/Users/matt/code/akka-http-docker/)
[info] Run completed in 48 milliseconds.
[info] Total number of tests run: 0
[info] Suites: completed 0, aborted 0
[info] Tests: succeeded 0, failed 0, canceled 0, ignored 0, pending 0
[info] No tests were executed.
[success] Total time: 1 s, completed 06-May-2016 21:46:32
```

I think this is because the tests are structured as JUnit tests?

Is either https://github.com/sbt/junit-interface required, or refactor the tests as ScalaTest tests, or am I missing the obvious way to run the tests?

**Comment by meucaa** [comment]
Had the same issue ;)

Le vendredi 6 mai 2016, mattinbits notifications@github.com a écrit :

> When I run 'sbt test' the code compiles correctly but no tests are run.
> 
> sbt test
> [info] Loading project definition from /Users/matt/code/akka-http-docker/project
> [info] Set current project to akka-http-docker (in build file:/Users/matt/code/akka-http-docker/)
> [info] Run completed in 48 milliseconds.
> [info] Total number of tests run: 0
> [info] Suites: completed 0, aborted 0
> [info] Tests: succeeded 0, failed 0, canceled 0, ignored 0, pending 0
> [info] No tests were executed.
> [success] Total time: 1 s, completed 06-May-2016 21:46:32
> 
> I think this is because the tests are structured as JUnit tests?
> 
> Is either https://github.com/sbt/junit-interface required, or refactor
> the tests as ScalaTest tests, or am I missing the obvious way to run the
> tests?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/jbrisbin/akka-http-docker/issues/3

## 

*Lucas VUILLIER *
lucas.vuillier@gmail.com / 06.35.52.28.82

**Comment by mattinbits** [comment]
Created two pull requests to solve this in different ways. Personally I prefer to use Scalatest but the Junit-Sbt fix is small and simple.

**Comment by jbrisbin** [comment]
Scalatest is the long-term solution. Merged #5.

Thanks!
